terraform/modules/spack/karpenter.tf

```diff
@@ -104,3 +104,46 @@ resource "kubectl_manifest" "karpenter_node_template" {
     helm_release.karpenter
   ]
 }
+
+locals {
+  pcluster_ami_id = {
+    "prod": "ami-00b2c701206072ffc"
+    "staging": "ami-0e2c34d361e37afa2"
```

Comment:
Note, these are custom "EKS-optimized" AMIs based on the AWS parallel cluster AMI. The packer code I used to build them is on this branch - https://github.com/mvandenburgh/amazon-eks-ami/tree/pcluster-ami

Reply:
If these pcluster runners are going to be a longer term thing, we may want to run this packer build in CI. The AMI images are dependent on kubernetes version, so left un-automated this is another thing we'll have to manually update when we get to the kubernetes upgrade.